index.html

```diff
@@ -1197,29 +1215,29 @@ <h2>
                 <li>[=Issue a haptic effect=] to the actuator with |type|,
                 |params|, and the |playEffectTimestamp|.
                 </li>
-                <li>When the effect completes, if
-                [=this=].{{GamepadHapticActuator/[[playingEffectPromise]]}} is
-                not `null`, [=queue a global task=] on the [=relevant global
-                object=] of [=this=] using the [=gamepad task source=] to run
-                the following steps:
+                <li>When the effect completes, [=queue a global task=] on the
+                [=relevant global object=] of [=this=] using the [=gamepad task
+                source=] to run the following steps:
                   <ol>
-                    <li>If
-                    [=this=].{{GamepadHapticActuator/[[playingEffectPromise]]}}
-                    is `null`, abort these steps.
-                    </li>
-                    <li>[=Resolve=]
-                    [=this=].{{GamepadHapticActuator/[[playingEffectPromise]]}}
-                    with {{GamepadHapticsResult/"complete"}}.
-                    </li>
-                    <li>Set
-                    [=this=].{{GamepadHapticActuator/[[playingEffectPromise]]}}
-                    to `null`.
+                    <li>If [=this=].{{GamepadHapticActuator/[[sequenceId]]}} is
```

Comment:
You could consider replacing the conditional nesting with an early return(example [usage](https://html.spec.whatwg.org/#dom-trees:parser-inserted-flag)).

Reply:
Done